src/QMCWaveFunctions/Fermion/SlaterDetOpt.cpp

```diff
@@ -518,17 +518,9 @@ void SlaterDetOpt::restore(int iat)
 /// \return  the log of the determinant value
 ///
 ///////////////////////////////////////////////////////////////////////////////////////////////////
-OrbitalBase::RealType SlaterDetOpt::registerData(ParticleSet& P, BufferType& buf) {
+void SlaterDetOpt::registerData(ParticleSet& P, WFBufferType& buf) {
```

Comment:
The return type is changed and is not consistent with the doxygen comment. One or both are wrong.